### DISCUSSION TITLE
slim down feature request template

### DISCUSSION THREAD

**Discussion by GhostOfTheSynthesis** [PrideCraft/Public-Issue-Tracker#8] [General]
I honestly think there's too much content in the feature request template. Most suggestions in the past were just a simple sentence or two, but this template makes you write out a whole book. I think we could just work with "username," "suggestion" and "any additional info" lines. Everything else is just excessive to me. 
I feel like if their feature request was related to a problem, they would initially say something in their suggestion without any prompt. I don't think anyone really thinks about alternatives to their suggestion, that's really up to us unless we need to follow up with the player.

**Comment by Notarin** [comment]
Locking, see issue #9
